src/harshadBJun20/CountOfCharsUsingWrapper.java

```diff
@@ -0,0 +1,47 @@
+package harshadBJun20;
+
+/*Assignment 17 : 25th Jun 2020
+
+1) Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2*/
+
+public class CountOfCharsUsingWrapper{
+	int digitCount,letterCount,uppercaseCount,lowercaseCount,specialCharCount;
+	void printCountOfChars(String str1) {
+		
+		for(int index=0;index<str1.length();index++){
+			if(Character.isDigit(str1.charAt(index))==true)
+				digitCount++;
+			else if(Character.isLetter(str1.charAt(index))==true)
+				letterCount++;
```

Comment:
letterCount without using this variable and else if block you can achieve expected output. Think about it.

Reply:
-> Changes done as per comments. Please review. Thanks.